doc/source/whatsnew/v0.24.0.rst

```diff
@@ -668,6 +669,35 @@ is the case with :attr:`Period.end_time`, for example
 
    p.end_time
 
+.. _whatsnew_0240.api_breaking.datetime_unique:
+
+The return type of :meth:`Series.unique` for datetime with timezone values has changed
+from an :class:`ndarray` of :class:`Timestamp` objects to a :class:`arrays.DatetimeArray` (:issue:`24024`).
+
+Previous Behavior:
```

Comment:
use bold here and below

Reply:
It looks like https://github.com/pandas-dev/pandas/pull/24298/files is standardizing on italics (single `*`), so I'll do that.